.github/workflows/wpt.yml

```diff
@@ -45,8 +51,8 @@ jobs:
         working-directory: wpt
       - name: Install Chromium
         run: echo "chromium_binary=$(npx @puppeteer/browsers install chromium@latest | cut -f 2- -d' ')" >> $GITHUB_ENV
-      - name: Run WPT tests
-        if: ${{ matrix.os == 'ubuntu-latest' && matrix.head == 'headful' }}
+      - name: Run WPT tests (mapper, headful)
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.head == 'headful' && matrix.kind == 'mapper' }}
```

Comment:
Headless doesn't depend on `xvfb` though, this would introduce a behavior change.

In the past we had `command prefix` as one of the `include:`s above, and by using it we could keep a single `run:` block.

Do you feel strong about this suggestion?